person.rb

```diff
@@ -7,12 +8,17 @@ def initialize(age:, name: 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @corrector = corrector
```

Comment:
To be able to create an instance of a any class we need to use the `new()` method 👍 

For example : ⬇️ 
 - `@animal= Animal.new()`
 
 More info [LINK](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/composition_by_examples.md#composable-classes)
 
 
 - [ ] Change `@corrector` instance variable to

Reply:
Done ✔